src/reactive/extMethods/do.ts

```diff
@@ -14,15 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- // notice_end
+// notice_end
 
-import Observable from '../Observable';
-import { Guard } from '../../system';
+import {Observable} from '../Observable';
+import {Guard} from '../../system';
 
-Observable.prototype.do = function(action) {
+Observable.prototype.do = function (action) {
```

Comment:
can type action

Reply:
Most of the extensionMethods can do with the typing. Won't add a comment on all of them